src/core/server/saved_objects/migrationsv2/integration_tests/migration_7_13_0_transform_failures.test.ts

```diff
@@ -122,11 +140,11 @@ describe('migration v2', () => {
       }
 
       expect(errorMessage.includes('7 transformation errors were encountered:')).toBeTruthy();
-      expect(
-        errorMessage.includes(
-          'space:default: Error: Document "default" has property "space" which belongs to a more recent version of Kibana [6.6.0]. The last known version is [undefined]'
```

Comment:
To be clear, this case is no longer possible in v2 migrations now, correct?

Reply:
Correct,  `The last known version is [undefined]` scenario  is no longer possible during the migration as it would now fail before trying to transform the documents